python/py-pyside6/Portfile

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           qt6 1.0
+
+name                py-pyside6
+revision            0
+categories-append   devel aqua
+
+maintainers         @johankytt openmaintainer
```

Comment:
This format is only for mambers with commit access. You'll want to specify your email like [this](https://github.com/macports/macports-ports/commit/90e47ab0d5f4b5a914694f430c47b3c762646a72#diff-a3cc3353201ba9de2f7cc6e7173c36f94ceb75cbdc9151697b2aa58c983ced67L11).